### ISSUE TITLE
Charger moins de polices de caractères

### ISSUE THREAD

**Issue by dumbbell** [AuxQuatreCoinsDeParis/aux-quatre-coins-de-paris.fr#2]
Si on regarde la liste des polices chargées depuis fonts.googleapis.com, il y en a plein qui sont certainement inutiles :
https://fonts.googleapis.com/css?family=Cabin+Sketch%3A%2C400%2C400italic%2C500%2C500italic%2C600%2C600italic%2C700%2C700italic%2C800%2C800italic%7CRoboto%3A300%2C300italic%2C400%2C400italic%2C500%2C500italic%2C600%2C600italic

Il y a des polices pour des caractères cyrilliques, vietnamiens, grècs, etc. Le tout dans trois tailles différentes (300, 400 et 500). Il y a certainement moyen d'optimiser ça :-)

**Comment by Aurelielb** [comment]
Hum... en fait j'ai déjà écrémé ce que je pouvais, mais je vais regarder
sur google font s'il n'y a pas des options pour éviter les caractères
cyrilliques, etc. ...

**Comment by dumbbell** [comment]
Je crois que je me trompe pour les caractères cyrilliques & friends. Ils sont bien listés dans la CSS générée par Google Fonts, mais je n'ai pas l'impression qu'ils soient chargés réellement. Ils doivent être chargés à la demande.

Je crois que le vrai problème est le nombre de graisses et styles qu'on utilise :
* Light (300), normal et italique
* Regular (400), normal et italique
* Medium (500) normal et italique
* Bold (700) normal et italique
(de mémoire, c'était ça)

Peut-être qu'on peut se limiter à deux graisses déjà (par exemple Light + Medium) ?

**Comment by Aurelielb** [comment]
Oui, mais on n'aura plus le même rendu sur certains textes / titres,
j'avais déjà fait des tests la dessus. En même temps on a changé des trucs
depuis alors je vais re-tester tout ça, mais je ne te promet rien :/
